src/Swarm/TUI/Model.hs

```diff
@@ -836,6 +840,7 @@ initUIState showMainMenu cheatMode = liftIO $ do
       , _uiReplHistory = newREPLHistory history
       , _uiReplLast = ""
       , _uiInventory = Nothing
+      , _uiInventorySort = InventorySortOptions Ascending ByNaturalAlphabetic
```

Comment:
I think we should also reset the `uiInventorySort` in `scenarioToUIState`, so the sort order always starts out as ascending alphabetic every time you start a game.  Currently, whatever the sort order was in your previous game persists into the next.